packages/rlc-common/src/buildLogger.ts

```diff
@@ -9,6 +9,10 @@ export function buildLogger(model: RLCModel) {
   if (!model.options) {
     return undefined;
   }
+  // Disable logger for non-branded packages
+  if (model.options.branded === false) {
```

Comment:
Could you explain why we need to disable logger for non-branded packages?

Reply:
Yes, logger is removed in ts http runtime.